darwin/exporter/formats/coco.py

```diff
@@ -377,6 +378,16 @@ def _build_image(annotation_file: dt.AnnotationFile, tag_categories: Dict[str, i
     tags = [
         annotation for annotation in annotation_file.annotations if annotation.annotation_class.annotation_type == "tag"
     ]
+
+    # CoCo file format requires unique image IDs
+    # darwin 1.0 produces unique 'seq' values that can be used
+    # darwin 2.0 does not provide `seq` so we hash the path + filename to produce a unique-enough 32bit int
+    if annotation_file.seq:
+        image_id = annotation_file.seq
+    else:
+        full_path = (annotation_file.remote_path or "") + annotation_file.filename
```

Comment:
```suggestion
        full_path = Path(annotation_file.remote_path or "/") / Path(annotation_file.filename)
```

Otherwise we glue both with no separator creating a case where (`/my/path`, `file`) is the same as `(/my/pathfi`, `le`)